models/User.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from "mongoose";
+
+const UserSchema = new mongoose.Schema(
+  {
+    username: {
+      type: String,
+      required: true,
+      maxlength: 30,
+    },
+    password: {
+      type: String,
+      required: true,
+      maxlength: 10,
+      minlength: 3,
```

Comment:
If this was a real project, both minlength and maxlength would have to be bigger.

Reply:
this is a real project and i agreee with @stratosfloor maybe change min and max length on password?